src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use `*` selector (it impacts performance). Set styles only for elements that require them. Instead, you can set `margin: 0` and `padding: 0` in the `body` tag if you need to reset the default browser styling.